### PR TITLE
Map merchant to Vendor in QBO

### PR DESCRIPTION
**Pull request by Shwetabhk**

### CODE REVIEW

File: apps/quickbooks_online/tasks.py [hunk 1]
```diff
@@ -28,6 +28,26 @@
 logger = logging.getLogger(__name__)
 
 
+def get_or_create_credit_card_vendor(workspace_id, merchant):
```
Comment (ashwin1111) [inline apps/quickbooks_online/tasks.py:31]: arg type :)

File: apps/workspaces/utils.py [hunk 2]
```diff
@@ -89,9 +95,21 @@ def create_or_update_general_settings(general_settings_payload: Dict, workspace_
...
+
+    if general_settings.map_merchant_to_vendor and \
+            general_settings.corporate_credit_card_expenses_object == 'CREDIT CARD PURCHASE':
+        expense_group_settings = ExpenseGroupSettings.objects.get(workspace_id=workspace_id)
+
+        ccc_expense_group_fields = expense_group_settings.corporate_credit_card_expense_group_fields
+        ccc_expense_group_fields.append('expense_id')
+        expense_group_settings.corporate_credit_card_expense_group_fields = ccc_expense_group_fields
+
+        expense_group_settings.save(update_fields=['corporate_credit_card_expense_group_fields'])
```
Comment (ashwin1111) [inline apps/workspaces/utils.py:111]: let's use .save() pliss

**Review by ashwin1111 — APPROVED** [review]

File: apps/quickbooks_online/tasks.py (hunk 1 above)
Comment (Shwetabhk) [reply apps/quickbooks_online/tasks.py:31]: will add

File: apps/workspaces/utils.py (hunk 2 above)
Comment (Shwetabhk) [reply apps/workspaces/utils.py:111]: changing

Comment (Sravanksk) [reply apps/workspaces/utils.py:111]: +1

**Review by Sravanksk — APPROVED** [review]

**Review by KirtiGautam — APPROVED** [review]

**Review by NileshPant1999 — APPROVED** [review]

**Pull request merged**